numpy/random/tests/test_direct.py

```diff
@@ -559,3 +559,22 @@ def test_passthrough(self):
         rg2 = default_rng(rg)
         assert rg2 is rg
         assert rg2.bit_generator is bg
+
+    def test_coercion_RandomState_Generator(self):
+        # use default_rng to coerce RandomState to Generator
+        rs = RandomState(1234)
+        rg = default_rng(rs)
+        assert isinstance(rg.bit_generator, MT19937)
+        assert rg.bit_generator is rs._bit_generator
+
+        # RandomState with a non MT19937 bit generator
+        _original = np.random.get_bit_generator()
+        bg = PCG64(12342298)
+        np.random.set_bit_generator(bg)
+        rs = np.random.mtrand._rand
+        rg = default_rng(rs)
+        assert rg.bit_generator is bg
```

Comment:
Ignorable nit: I'd probably just delete this section or construct a fresh `RandomState` instance with the specified `bg` if you really want to test a non-MT19937 `BitGenerator` (e.g. `rs = RandomState(bg)`). Mucking with the global `RandomState` instance isn't testing anything about `default_rng()`.